Episode-Proposal-Template

```diff
@@ -0,0 +1,37 @@
+Episode Proposal Template
+
+# [TITLE]
+
+* Proposal: [PR#n]([link to PR])
```

Comment:
Would this need an associated pull request, or will it just be an issue?

Reply:
Or are you thinking to have all proposals be a pull request, like with swift-evolution?

Reply:
I think all proposals would be a PR and that the proposal would be updated to include the PR# and link to PR. Might be overkill but I will try and see. what do you think?

Reply:
I think that sounds good!